core-plugins/docs/PythonEvaluator-transform.md

```diff
@@ -25,35 +25,40 @@ schema is the same as the input schema.
 
 Example
 -------
-The transform takes records that have a ``'subtotal'`` field, calculates ``'tax'`` and
-``'total'`` fields based on the subtotal, and then returns a record, as a Python dictionary,
-containing those three fields:
+The transform takes records that have a ``'subtotal'`` field, if ``'subtotal'`` is negative, it emits an error,
+else it calculates ``'tax'`` and ``'total'`` fields based on the subtotal, and then returns a record,
+as a Python dictionary, containing those three fields:
```

Comment:
The transform checks each record's ``'subtotal'`` field: if the ``'subtotal'`` is negative, it emits an error;
else, it calculates the ``'tax'`` and ``'total'`` fields based on the ``'subtotal'``, and then returns a record
as a Python dictionary containing those three fields: